ArduSub/ArduSub.cpp

```diff
@@ -344,4 +346,15 @@ void Sub::update_altitude()
     }
 }
 
+void Sub::update_strobe_light()
+{
+    if(RC_Channels::get_radio_in(11) < 1500) {
+        return;
+    }
+
+    static bool strobe_state;
+    hal.rcout->write(9, strobe_state ? 1100 : 1900);
```

Comment:
Brightness and strobe state should be independent of each other. We should be writing the currently selected brightness level.